src/main/java/uk/gov/justice/digital/delius/config/ApplicationConfig.java

```diff
@@ -20,17 +20,20 @@
 public class ApplicationConfig {
 
     @Bean
-    public MappingJackson2HttpMessageConverter mappingJackson2HttpMessageConverter() {
+    public MappingJackson2HttpMessageConverter mappingJackson2HttpMessageConverter(ObjectMapper objectMapper) {
         MappingJackson2HttpMessageConverter jsonConverter = new MappingJackson2HttpMessageConverter();
-        ObjectMapper objectMapper = new ObjectMapper()
-                .configure(DeserializationFeature.FAIL_ON_UNKNOWN_PROPERTIES, false)
+        customiseObjectMapper(objectMapper);
+        jsonConverter.setObjectMapper(objectMapper);
+        return jsonConverter;
+    }
+
+    public static ObjectMapper customiseObjectMapper(final ObjectMapper objectMapper) {
```

Comment:
We can now call this method from unit tests where it is not possible to autowire the object mapper.